.github/actions/javascript/getDeployPullRequestList/getDeployPullRequestList.ts

```diff
@@ -40,10 +40,15 @@ async function run() {
                 })
             ).data.jobs.some((job) => job.name.startsWith('Build and deploy') && job.conclusion === 'success')
         ) {
+            console.log(`Deploy was not a success: ${lastSuccessfulDeploy.html_url}, looking at the next one`);
             lastSuccessfulDeploy = completedDeploys.shift();
         }
 
-        const priorTag = completedDeploys[0].head_branch;
+        if (!lastSuccessfulDeploy) {
+            throw new Error('Could not find a prior successful deploy');
```

Comment:
This doesn't really seem possible, but it was needed to make TS happy because `lastSuccessfulDeploy` was potentially undefined